robottelo/performance/candlepin.py

```diff
@@ -29,7 +29,7 @@ class Candlepin(object):
     serverhost = conf.properties['main.server.hostname']
 
     @classmethod
-    def get_real_time(result):
+    def get_real_time(cls, result):
```

Comment:
@Ichimonji10, in fact, this is for bug-fixing. I previously missed the cls there.